scheds/rust/scx_layered/src/main.rs

```diff
@@ -3323,6 +3327,27 @@ fn expand_template(rule: &LayerMatch) -> Result<Vec<(LayerMatch, Cpumask)>> {
                 )
             })
             .collect()),
+        LayerMatch::CgroupRegex(expr) => Ok(traverse_sysfs(Path::new("/sys/fs/cgroup"))?
```

Comment:
At some point we should get the mount point with [`get_fs_mount`](https://github.com/sched-ext/scx/blob/6b4fcd28b667b3184f37106a310d4804c4c1f304/rust/scx_utils/src/compat.rs#L183).